build/gen_stub.php

```diff
@@ -629,26 +662,332 @@ private function getFlagsAsString(): string
 
         return $flags;
     }
+}
+
+class PropertyInfo
+{
+    /** @var PropertyName */
+    public $name;
+    /** @var int */
+    public $flags;
+    /** @var bool */
+    public $isKnownName;
+    /** @var bool */
+    public $isDeprecated;
```

Comment:
I'm not sure we should be storing this information. We currently can't deprecate properties, right?